### PR TITLE
Contact form

### LINKED ISSUE #21: Contact Form
# Design
Contact Form

# To Do
Form with: 
- [ ] email
- [ ] first and last name
- [ ] message 
- [ ] send button from #12

### PR DESCRIPTION
**Pull request by danifosy**
Will resolve #21 
https://a-dogs-life-contactform-e0epyg.herokuapp.com/

### CODE REVIEW

File: src/app/pages/ContactForm/ContactForm.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import TextInput from '../../components/TextInput/TextInput';
+import SubmitButton from '../../components/SubmitButton/SubmitButton';
+import styles from './ContactForm.module.css';
+import Typography from '../../components/Typography/Typography';
+Typography;
```
Comment (lajanzen) [inline src/app/pages/ContactForm/ContactForm.tsx:6]:
```suggestion
```

File: src/app/pages/ContactForm/ContactForm.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import TextInput from '../../components/TextInput/TextInput';
+import SubmitButton from '../../components/SubmitButton/SubmitButton';
+import styles from './ContactForm.module.css';
+import Typography from '../../components/Typography/Typography';
```
Comment (lajanzen) [inline src/app/pages/ContactForm/ContactForm.tsx:5]: Do you still need this?

**Review by lajanzen — APPROVED** [review]
Nice! Maybe you might want to add a file ContactForm.stories.tsx to add a story on storybook? 😉

File: src/app/pages/ContactForm/ContactForm.tsx [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+Typography;
+
+export default function ContactForm(): JSX.Element {
+  return (
+    <main>
+      {/* <h2>
+        <Typography size="l">Kontaktanfrage</Typography>
+      </h2> */}
```
[the remark this answers is not in the document]
Comment (danifosy) [reply src/app/pages/ContactForm/ContactForm.tsx:13]: I'm indecisive  🙈

File: src/app/pages/ContactForm/ContactForm.tsx [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+      <form className={styles.contactForm}>
+        <TextInput inputType="text" id="Vor-und-nachname">
```
[the remark this answers is not in the document]
Comment (danifosy) [reply src/app/pages/ContactForm/ContactForm.tsx:15]: Yea that sounds good, thank you!

**Pull request merged**